core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java

```diff
@@ -148,8 +153,26 @@ private void saveDlpPositiveItem(Node node) throws RepositoryException {
     }
     dlpPositiveItemEntity.setType(TYPE);
     dlpPositiveItemEntity.setDetectionDate(Calendar.getInstance());
-    // to be updated with detected keyword
-    dlpPositiveItemEntity.setKeywords(dlpKeywords);
+    dlpPositiveItemEntity.setKeywords(getDetectedKeywords());
     dlpPositiveItemService.addDlpPositiveItem(dlpPositiveItemEntity);
   }
+
+  private String getDetectedKeywords() {
```

Comment:
Please add a test case to validate this function result

Reply:
Done !